src/components/app/layout/Header.module.scss

```diff
@@ -9,11 +9,29 @@
   .logo-container {
     width: 90px;
     height: 30px;
+    margin-left: 50px;
 
     .navbar-logo {
       width: 100%;
       height: 100%;
       object-fit: contain;
     }
   }
+  .links-container {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    margin-right: 50px;

```

Comment:
we don't need to add margin here because we already applied padding to the big container (the nav element)